src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java

```diff
@@ -126,6 +126,7 @@
     public static final String MAPPING_QUALITY_ZERO_BY_SAMPLE_KEY = "MQ0";
     public static final String HAPLOTYPE_CALLER_PHASING_GT_KEY =    "PGT";
     public static final String HAPLOTYPE_CALLER_PHASING_ID_KEY =    "PID";
+    public static final String HAPLOTYPE_CALLER_PHASING_SET_KEY =   "PS";
```

Comment:
This is already in the htsjdk VCFConstants because it's reserved in the VCF spec.